packages/laverna/README.md

```diff
@@ -78,7 +82,11 @@ If you're publishing a package in a new workspace, you might:
 
 ### Automating Publishes
 
-You'll need to figure out how to add `--newPkg` to your CI/CD pipeline. Otherwise, use the `--yes` flag when invoking `laverna`.
+If the `CI` env var is present, `laverna` will skip the confirmation prompt before final publish (i.e., `--yes` defaults to `true`).
+
+#### Automating Publishes of New Packages
```

Comment:
Mixed feelings here - the tool exists because we wanted to avoid automated publishing and now we're endorsing it. I'd slap a big orange warning here saying automated publishing is a 2FA bypass.

Reply:
agree.  I just wanted the feature for my own stuff